tests/scripts/thread-cert/node.py

```diff
@@ -2008,12 +2037,22 @@ def ping_ether(self, ipaddr, num_responses=1, size=None, timeout=5, ttl=None) ->
         return resp_count
 
     def _getBackboneGua(self) -> Optional[str]:
-        for ip6Addr in self.get_addrs():
-            if re.match(config.BACKBONE_PREFIX_REGEX_PATTERN, ip6Addr, re.I):
-                return ip6Addr
+        for addr in self.get_addrs():
+            if re.match(config.BACKBONE_PREFIX_REGEX_PATTERN, addr, re.I):
+                return addr
 
         return None
 
+    def _getInfraUla(self) -> Optional[str]:
+        """ Returns the ULA addresses autoconfigured on the infra link.
+        """
+        addrs = []
+        for addr in self.get_addrs():
+            if re.match(config.ONLINK_PREFIX_REGEX_PATTERN, addr, re.I):
```

Comment:
mesh local addresses might also start with `fd` ?

Reply:
Sure, but this is used for matching host's  addresses.